src/main/java/com/minecolonies/colony/permissions/Permissions.java

```diff
@@ -173,21 +173,21 @@ public static Rank getDemotionRank(Rank rank)
     }
 
     /**
-     * Reads the permissions from a NBT
+     * Reads the permissionMap from a NBT.
      *
-     * @param compound NBT to read from
+     * @param compound NBT to read from.
      */
-    public void loadPermissions(@NotNull NBTTagCompound compound)
+    public void loadPermissions(@NotNull final NBTTagCompound compound)
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) The Cyclomatic Complexity of this method "loadPermissions" is 11 which is greater than 10 authorized. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AMethodCyclomaticComplexity)